optimism/material/HyperViscoelastic.py

```diff
@@ -109,108 +66,40 @@ def _eq_strain_energy(dispGrad, props):
     return Wdev + Wvol
 
 def _neq_strain_energy(dispGrad, stateOld, dt, props):
-    I = np.identity(3)
-    F = dispGrad + I
-    state_new = _compute_state_new(dispGrad, stateOld, dt, props)
-    # Fvs = state_new.reshape((NUM_PRONY_TERMS, 3, 3))
-    Fv_new = state_new.reshape((3, 3))
-
     G_neq = props[PROPS_G_neq]
     tau   = props[PROPS_TAU]
     eta   = G_neq * tau
 
-    Fe = F @ np.linalg.inv(Fv_new)
-    Ee = 0.5 * TensorMath.mtk_log_sqrt(Fe.T @ Fe)
+    Ee_trial = _compute_elastic_logarithmic_strain(dispGrad, stateOld)
+    state_inc = _compute_state_increment(Ee_trial, dt, props)
```

Comment:
We could give this variable a clearer name like `viscous_strain_increment`, or `delta_Ev`, or something else like that. The J2 model uses `state_inc` because it's a tuple of things.